content/en/hosting/4.x/_partial_values_explanation.md

```diff
@@ -0,0 +1,12 @@
+---
+toc_hide: true
+hide_summary: true
+---
+
+* `<your-namespace>` (_two occurrences_)
+* `<version>` - 4.x version you're upgrading to
+* `<password>` - retrieved from `get-env` call above
+* `<secret>` - retrieved from `get-env` call above
+* `<admin_user>` - needs to be the same as used in 3.x - likely `medic`
+* `<uuid>` - retrieved from `get-env` call above
+* `<url>` - your DNS goes here (your-dns-address.org)
```

Comment:
```suggestion
---

Be sure to update the following values in your YAML file:

* `<your-namespace>` (_two occurrences_)
* `<version>` - 4.x version you're upgrading to
* `<password>` - retrieved from `get-env` call above
* `<secret>` - retrieved from `get-env` call above
* `<admin_user>` - needs to be the same as used in 3.x - likely `medic`
* `<uuid>` - retrieved from `get-env` call above
* `<url>` - the URL of your production instance goes here (eg `www.example.org`)
* `<path-to-tls>` - path to TLS files on disk
```

Reply:
I have removed the www and used just example.com.